manager/assets/modext/widgets/security/modx.panel.groups.roles.js

```diff
@@ -69,6 +80,7 @@ Ext.extend(MODx.panel.GroupsRoles,MODx.FormPanel,{
                             ,useSplitTips: true
                             ,monitorResize: true
                             ,width: 270
+                            ,minWidth: 270
                             ,minSize: 270
                             ,maxSize: 400
```

Comment:
I know that it was here before but could it be increased to half of width of view? Not important, just interesting.